st3/sublime_lib/testing.py

```diff
@@ -0,0 +1,19 @@
+import sublime
+
+from unittest import TestCase
+
+class ViewTestCase(TestCase):
+    def setUp(self):
+        self.view = sublime.active_window().new_file()
+
+    def tearDown(self):
+        if self.view:
+            self.view.set_scratch(True)
+            self.view.window().focus_view(self.view)
+            self.view.window().run_command("close_file")
```

Comment:
Isn't `close_file` a view command? If so, you could just run `self.view.run_command("close_file")`.